src/sentry/utils/pytest/sentry.py

```diff
@@ -15,16 +15,7 @@
 def pytest_configure(config):
     import warnings
 
-    from django.utils.deprecation import RemovedInDjango20Warning, RemovedInDjango21Warning
-
-    # These warnings should be kept in sync with sentry.runner.settings,
-    # and pytest warningfilters in pyproject.toml.
-    # See pyproject.toml for explanations.
-    warnings.filterwarnings(action="ignore", category=RemovedInDjango20Warning)
-    warnings.filterwarnings(action="ignore", category=RemovedInDjango21Warning)
```

Comment:
This stuff isn't needed, it doesn't affect pytest failing; it only yells about stuff before pytest session starts which is just a cosmetic issue at best.